aten/src/ATen/native/PixelShuffle.cpp

```diff
@@ -62,6 +67,10 @@ Tensor pixel_shuffle_cpu(const Tensor& self, int64_t upscale_factor) {
 Tensor pixel_unshuffle_cpu(const Tensor& self, int64_t downscale_factor) {
   check_pixel_unshuffle_shapes(self, downscale_factor);
 
+  if (self.numel() == 0) {
+    return self.clone();
+  }
```

Comment:
Why do we need to clone `self` in this case? Isn't returning `self` just enough?

Reply:
I tried but without cloning `self` it encounter the following error : `RuntimeError: Argument input is not defined to alias output but was aliasing`.

Reply:
fair enough